### DISCUSSION TITLE
Couldn't able to load amt and raster package in R

### DISCUSSION THREAD

**Discussion by KathanEcol1** [jmsigner/amt#70] [General]
Even if I tried to install the packages from repository using the following code using 
install.packages('raster', version = "1.5-12", repos='https://rspatial.r-universe.dev') 
or,
install.packages('raster', repos='https://rspatial.r-universe.dev')

I can not load the package and the same error is coming up like:

"Error: package or namespace load failed for ‘raster’ in get0(".packageName", env, inherits = FALSE):
 lazy-load database '/Library/Frameworks/R.framework/Versions/4.2/Resources/library/raster/R/raster.rdb' is corrupt
In addition: Warning message:
In get0(".packageName", env, inherits = FALSE) :
  internal error -3 in R_decompress1"

By the way I am using a Macbook!

**Comment by jmsigner** [comment]
Hi, 

this looks like an issue with the `raster` package to me. Can you load the raster package? Some discussion about this is [here](https://gis.stackexchange.com/questions/422328/raster-package-libraryraster-doesnt-work). 

Sorry for not having better suggestions
